test/state/bloom.hpp

```diff
@@ -0,0 +1,43 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "hash_utils.hpp"
+#include <intx/intx.hpp>
+#include <span>
+
+namespace evmone::state
+{
+struct TransactionReceipt;
+struct Log;
+
+/// The 2048-bit hash suitable for keeping an Ethereum bloom filter value.
+struct BloomFilter
+{
+    /** The 256 bytes of the bloom filter value. */
+    uint8_t bytes[256];
+
+    constexpr BloomFilter() : bytes{} {}
+
+    explicit BloomFilter(const evmone::bytes& data) : bytes{}
```

Comment:
Where do you need it. Would be very good to drop it.

Reply:
I did it but now in tests:
```
const auto bloom_4 = "0x0000000000000000000000000000000000000000000000000000000000000"
        "000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000"
        "000100000000000000000000040000000000000000000000040000000000000000000000000000000000000000"
        "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
        "000000000000000000000000000000000000000000000000000000000040000000000000000200000000000000"
        "000000000000600000000000000000000000000000000000000000000000000000000000001000000000000000"
        "0"_hex;

    TransactionReceipt r4;
    memcpy(r4.logs_bloom_filter.bytes, bloom_4.data(), 256);
```